tests/test_bgp_af.rb

```diff
@@ -745,6 +544,7 @@ def default_metric(asn, vrf, af)
   def test_default_metric
     afs = [%w(ipv4 unicast), %w(ipv6 unicast)]
     afs.each do |af|
+      config_ios_xr_dependencies('55') if platform == :ios_xr
```

Comment:
Ideally both should work (string is sometimes needed to support ASDOT notation) but it's not critical especially for this PR. Just seemed inconsistent that this function takes `'55'` while the function below takes `55`. Not blocking.